internal/backends/dotnet/dotnet.go

```diff
@@ -11,7 +11,7 @@ var DotNetBackend = api.LanguageBackend{
 	Name:             "dotnet",
 	Specfile:         findSpecFile(),
 	Lockfile:         lockFileName,
-	FilenamePatterns: []string{"*.cs", "*.csproj"}, // C# support so far
+	FilenamePatterns: []string{"*.cs", "*.csproj", "*.fs", "*.fsproj"}, // C# support so far
```

Comment:
You might want to remove the comment there

Reply:
Removed the comment